lldb/include/lldb/Host/ProcessRunLock.h

```diff
@@ -29,8 +29,13 @@ class ProcessRunLock {
 
   bool ReadTryLock();
   bool ReadUnlock();
+
+  /// Set the process to running. Returns true if the process was stopped.
+  /// Return false if the process was running.
   bool SetRunning();
-  bool TrySetRunning();
+
+  /// Set the process to stopped. Returns true if the process was stopped.
+  /// Returns false if the process was running.
```

Comment:
```suggestion
  /// Set the process to stopped. Returns true if the process was running.
  /// Returns false if the process was stopped.
```

(at least, that's what the windows implementation does -- and I think it makes sense)